en/user/articles-dxp/120-search/12-synonyms.markdown

```diff
@@ -46,6 +46,22 @@ The `=>`
 [format](https://www.elastic.co/guide/en/elasticsearch/guide/current/synonym-formats.html)
 supported in Elasticsearch is not supported through the Synonyms Set UI.
 
```

Comment:
@mtokudome, this info (feel free to reorganize and adjust) can go into the synonyms article. :point_down: